### DISCUSSION TITLE
About the actual need and solutions for a formal specification

### DISCUSSION THREAD

**Discussion by ericjenn** [ericjenn/working-groups#4] [General]
What do we actually need in terms of specification level, specification formalism? 
Do we really need a _formal specification language_ (Coq, WhyMl, PVS, Z, etc.) or can we live with something lighter?

**Comment by jeansouyris** [comment]
If no formal language is used, the "lighter" format shall be sufficiently accurate, consistent and non-ambigous to enable one to derive a specification in a formal language from it.
